hack/label_sync/labels.yaml

```diff
@@ -18,91 +18,244 @@ default:
       addedBy: prow
 repos:
   3scale-ops/marin3r:
     labels:
-      # Test
-      - color: b60205
-        description: Indicates a PR that requires an org member to verify it is safe to test.
-        name: needs-ok-to-test
-        target: prs
-        prowPlugin: trigger
-        addedBy: prow
-      - color: 15dd18
-        description: Indicates a non-member PR verified by an org member that is safe to test.
-        name: ok-to-test
-        target: prs
-        prowPlugin: trigger
-        addedBy: prow
+    # Test
+    - color: b60205
+      description: Indicates a PR that requires an org member to verify it is safe to test.
+      name: needs-ok-to-test
+      target: prs
+      prowPlugin: trigger
+      addedBy: prow
+    - color: 15dd18
+      description: Indicates a non-member PR verified by an org member that is safe to test.
+      name: ok-to-test
+      target: prs
+      prowPlugin: trigger
+      addedBy: anyone
+  3scale/3scale-saas:
+    labels:
+    # Kind
+    - name: kind/deploy
+      description: Categorizes issue or PR as related to a deploy.
+      color: 3CAB6C
+      target: both
+      addedBy: anyone
+    # Env
+    - name: env/common
+      description: Categorizes issue or PR as related shared across environments.
+      color: 34828C
+      target: both
+      addedBy: anyone
+    - name: env/dev-eng
+      description: Categorizes issue or PR as related to the dev-env environment.
+      color: C2E0C6
+      target: both
+      addedBy: anyone
+    - name: env/stg-saas
+      description: Categorizes issue or PR as related to the stg-saas environment.
+      color: FBCA04
+      target: both
+      addedBy: anyone
+    - name: env/pro-base
+      description: Categorizes issue or PR as related to the pro-base environment.
+      color: EE0000
+      target: both
+      addedBy: anyone
+  3scale/platform:
+    labels:
+    # Kind
+    - name: kind/maintenance
+      description: Categorizes issue or PR as related to maintenance tasks.
+      color: c7def8
+      target: both
+      addedBy: anyone
+    - name: kind/deploy
+      description: Categorizes issue or PR as related to a deploy.
+      color: 3CAB6C
+      target: both
+      addedBy: anyone
+    - name: kind/incident
+      description: Categorizes issue or PR as related to an incident, outage or postmortem.
+      color: b328d6
+      target: both
+      addedBy: anyone
+    # Env
+    - name: env/legacy
+      description: Categorizes issue or PR as related to the legacy environment.
+      color: 835C3B
+      target: both
+      addedBy: anyone
+    - name: env/common
+      description: Categorizes issue or PR as related shared across environments.
+      color: 34828C
+      target: both
+      addedBy: anyone
+    - name: env/dev-eng
+      description: Categorizes issue or PR as related to the dev-env environment.
+      color: C2E0C6
+      target: both
+      addedBy: anyone
+    - name: env/stg-saas
```

Comment:
Can you add the new label env/pro-saas?

Reply:
As discussed over slack, you will add the new label to validate the documentation and current procedure.